vizro-core/src/vizro/models/_navigation/nav_bar.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+
+import itertools
+from typing import List, Optional
+
+from dash import html
+from pydantic import validator
+
+from vizro.managers import model_manager
+from vizro.models import VizroBaseModel
+from vizro.models._models_utils import _log_call
+from vizro.models._navigation.nav_item import NavItem
+from vizro.models.types import NavPagesType
+
+
+class NavBar(VizroBaseModel):
+    """Navigation bar to be used as a selector for `Navigation`.
+
+    Args:
+        pages (Optional[NavPagesType]): See [`NavPagesType`][vizro.models.types.NavPagesType].
+            Defaults to `None`.
+        items (List[NavItem]): List of NavItem models. Defaults to `[]`.
```

Comment:
It would be great if we could keep the same docstring style. See other models for reference. Please double-check in other places as well

```suggestion
        items (List[NavItem]): See [`NavItem`][vizro.models.NavItem]. Defaults to `[]`.
```